cmd/vela-docker/plugin_test.go

```diff
@@ -34,8 +34,8 @@ func TestDocker_Plugin_Exec_BadWrite(t *testing.T) {
 		},
```

Comment:
I believe the name of the directory containing this file needs to be renamed 👍 

Currently it's:

`cmd/vela-docker/plugin_test.go`

When I believe it should be:

`cmd/vela-kaniko/plugin_test.go`